spec/models/request_item_spec.rb

```diff
@@ -4,24 +4,54 @@
 RSpec.describe RequestItem, type: :model do
   context "Methods >" do
     describe "#from_json" do
-      let(:organization) { build :organization }
-      let(:request) { build :request, organization: organization }
-      let(:request_item_json) { request.request_items.first }
-      let(:item) { Item.find(request_item_json['item_id']) }
+      let(:organization) { create :organization }
+      let(:location) { create :storage_location, organization: organization }
+      let(:other_location) { create :storage_location, organization: organization }
+      let(:request) { build :request, organization: organization, request_items: request_item_json }
+      let(:item1) { create(:item, organization: organization) }
+      let(:item2) { create(:item, organization: organization) }
+      let(:request_item_json) do
+        [
+          {item_id: item1.id, quantity: 15}.stringify_keys,
+          {item_id: item2.id, quantity: 18}.stringify_keys
+        ]
+      end
+
+      subject { described_class.from_json(request_item_json.first, organization, location) }
```

Comment:
It looks like the `from_json` only needs `json` and a `request`.  Based on line 4 of request item -- `def self.from_json(json, request)` I think you should just need to change the line to: `subject { described_class.from_json(request_item_json.first, request) }`